### ISSUE TITLE
A11y_30days_Welcome to 30days of learning_Reflow: When reflow is performed in "Menu hamburger icon" there is “Back to main menu" button which is not accessible through keyboard and mouse also.

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [microsoft/30daysof#171]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yTCS;#A11yMAS;#A11ySev2;#30days;#P3_30days_Dec22;#A11yWCAG2.1;#WCAG1.4.10;#DesktopWeb;#Win11;ChromiumEdge

### Environment Details:
Application Name: 30days
#URL: https://microsoft.github.io/30daysof/ 
Windows Version: Win11
Chromium Edge Version: 110.0.1556.0 (Official build) dev (64-bit)
### Prerequisite:
1.	Press "Ctrl+Shift+I" to open developer tool.
2.	Now in dock side click on "undock into separate window" so that developer tool will get open in separate window.
3.	Zoom the page to 175%.
4.	Set the screen resolution to 320px*256px.
### Repro Steps:
1.	Hit the URL: https://microsoft.github.io/30daysof/
2.	“30Days of Learning” page will open.
3.	Tab to “Menu hamburger” button and press enter.
4.	Tab to “Back to main menu” button and press enter and try to access with mouse.
5.	Observe that “Back to main menu” button is not accessible through keyboard and mouse.
### Actual:
When reflow is performed “Back to main menu” button present in “Menu hamburger icon” is not accessible through keyboard and mouse.
Note: This issue is also observed when “Resize” is performed.
### Expected:
The “Back to main menu” button present in “Menu hamburger icon” should be accessible through keyboard and mouse.
### User Impact:
Keyboard only users will face difficulty if control is not accessible through keyboard and users need to press additional keystrokes to go out of menu.

### Attachment:
[209321178-344db31a-5cd5-4a42-9923-45585c1b8bfb (1).webm](https://user-images.githubusercontent.com/85863895/211752179-5e7ff082-b933-41d3-8f1f-6ee9f4cf11da.webm)

**Comment by nitya** [comment]
@Ashutoshsahoo164 can you test if this issue is still present?

**Comment by Ashutoshsahoo164** [comment]
Hi @nitya Yes the issue is still present.

**Comment by nitya** [comment]
@Ashutoshsahoo164 we have decided to unpublish the site and deactivate it from internal listings. Can you verify and close this issue now that website is no longer active?

**Comment by Ashutoshsahoo164** [comment]
As the Site is now not be published , closing this bug.
